CustomApps/lyrics-plus/ProviderNetease.js

```diff
@@ -147,6 +147,37 @@ const ProviderNetease = (function () {
 		return lyrics;
 	}
 
+	function getTranslation(list) {
+		const lyricStr = list?.tlyric?.lyric;
```

Comment:
Does this only gets Chinese translations? Or any translations there is of the song.

Reply:
Only Chinese translation provide by netease music website

Reply:
Fair enough ig, the issue only showcased Netease music displaying Chinese translations

Reply:
Thank you, sir!